src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -13,13 +13,13 @@ if [ ! -d "$__PROJECT_DIR" ]; then
    exit 1
 fi
 
-if [ ! -d "$__TOOLS_DIR" ]; then
-   echo "ERROR: Cannot find tools path at '$__TOOLS_DIR'. Please pass in the tools directory as the 2nd parameter."
+if [ ! -e "$__DOTNET_CMD" ]; then
+   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 2nd parameter."
    exit 1
 fi
 
-if [ ! -e "$__DOTNET_CMD" ]; then
-   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 3rd parameter."
+if [ ! -d "$__TOOLRUNTIME_DIR" ]; then
```

Comment:
The script does `cp -R $__TOOLS_DIR/* $__TOOLRUNTIME_DIR` before calling dotnet publish.  The `cp` command won't work correctly if `__TOOLRUNTIME_DIR` doesn't already exist, so if we want to support the case where `__TOOLRUNTIME_DIR` doesn't exist, then I think we need to have init-tools.sh explicitly create the directory before copying to it.

The previous logic checked if the `__TOOLS_DIR` existed, but it referred to it as the 2nd parameter, so I assumed that it was intended to be checking the if the `__TOOLRUNTIME_DIR` existed.  Maybe that's not correct, though.  In any case, do you think I should add a `mkdir` before the `cp`?  Personally I'd be fine with either approach.